### ISSUE TITLE
Where are the CFML communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/cfml#82]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is CFML supported by one or more large organizations?
* Does CFML have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about CFML? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for CFML? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in CFML? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in CFML?
* Are there popular newsletters for CFML?
* Is CFML taught at programming bootcamps? (If so, what are some examples?)
* Is CFML taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about CFML?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by bdw429s** [comment]
> * Is CFML supported by one or more large organizations?

- Adobe - Owner of ColdFusion Server
- Lucee Association Switzerland (LAS) - Owner of FOSS Lucee Server

> * Does CFML have an official community manager?

- Elishia Dvorak at Adobe is the Product Evangelist
- Kishore Balakrishnan at Adobe is the community manager.  
- Lucee doesn't have any such official titles as of yet but there are quite a few volunteer community leaders.

> * Do you know of specific communities (online or offline) that are enthusiastic about CFML? (Chat communities, forums, meetups, student clubs, etc)

- All the CFUGs (ColdFusion User Groups) which can be found on Adobe's site https://coldfusion.adobe.com/user-groups/
- CFML Slack Team (~2.3k members) https://cfml-slack.herokuapp.com/
- "CFML Programmers" and "ColdFusion Programmers" Facebook groups
- CFML Programmers LinkedIn group
- Support forum for Lucee Server - https://dev.lucee.org
- ColdFusion Community Portal - https://coldfusion.adobe.com/

> * Are there popular conferences for CFML? (If so, what are some examples?)

- Adobe ColdFusion Summit - https://cfsummit.adobeevents.com/
- Ortus Into The Box - https://www.intothebox.org/
- NCDevcon - http://www.ncdevcon.com/
- CFCamp - https://www.cfcamp.org/
- MuraCon - https://2018.muracon.com/

> * Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in CFML? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)

Adobe has an initiative to get CFML taught in universities, but I'm not aware of any specific groups for tech minorities, though some CF conferences have had so-called "diversity passes" before.

> * Are there specific groups or programs dedicated to mentoring people in CFML?

The best one would be the CFML Slack team.  There's a channel in there dedicated to CFML beginners.

> * Are there popular newsletters for CFML?

Most CF-centric newsletters are driven by commercial entities in the CF Space.  
- Ortus Solutions mailing list - Sign up here: https://www.ortussolutions.com/
- Lucee Server mailing list - Sign up here: http://lucee.org/
- I know there's several more, but for the life of me I can't find newsletter signups on several of the company sites I was looking at.

You didn't ask about Podcasts, but I would consider them a more modern form of newsletters.  
- CF Alive Podcast - http://teratech.com/podcast/
- localhost Podcast - https://localhost.fm/

> * Is CFML taught at programming bootcamps? (If so, what are some examples?)

Sadly, not that I'm aware of.  

> * Is CFML taught at universities? (If so, what are some examples?)

Yes, there are a few but I don't know them off the top of my head.

**Comment by kytrinyx** [comment]
This is fantastic, thank you so much!

**Comment by kytrinyx** [comment]
I've got what I need for now; closing this out. Thank you! 🌷
